README.md

```diff
@@ -123,3 +123,52 @@ or run the Flutter application.
 `flutter run`
 
 > **NOTE**: A device with USB debugging enabled or virtual device is required.
+<br/>
+
+## For setting up the Flask backend
+### Technologies required
+
+-   **[Python3](https://www.python.org/downloads/)** - Make sure you have a python3 environment setup.
+-   **[Virtualenv](https://virtualenv.pypa.io/en/stable/)** - Make sure you install your packages locally in a virtual env and after adding a packing make sure to use 
+```bash
+pip freeze>requirements.txt
+```
+-   **[MySQL](https://www.mysql.com/)** - Preferrably MySQL-5.7 database is used. 
+-   **[SQLAlchemy](https://www.sqlalchemy.org)** - An SQL ORM
+<br/>
+
+1. Make a database in your MYSQL 
+   ```bash
+   CREATE {DATABASE | SCHEMA} [IF NOT EXISTS] labellab
+   ```
+2. Create a folder called called venv using 
+   ```bash
+   cd labellab-flask && mkdir venv
```

Comment:
When we create virtualenv, venv automatically gets created. We don't need to manually create this directory.

Reply:
okay sure I will change this thanks for pointing this out :)